templates/views/default-thread.html

```diff
@@ -2,11 +2,9 @@
     <div class="welcome">
         <img src="/@static/images/logo_text_under.svg" class="logo"/>
         <h2>Web Messenger</h2>
-        <h4>
-            <a href="https://youtu.be/GqCc-vgjTfA" target="_blank">
-                Learn how to get started
-                <i class="icon youtube large red"></i>
-            </a>
-        </h4>
+        <a data="videoModal" class="ui animated fade basic button">
```

Comment:
data tags should have a dash-property suffix, like `data-label="value"`.  See https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes